pkg/tmutil/tmutil.go

```diff
@@ -101,6 +102,19 @@ func checkPath(path string, verbose int) (bool, error) {
 	return !strings.Contains(s, "[Excluded]"), nil
 }
 
+func printTmutilError(path string, out bytes.Buffer, outErr bytes.Buffer) {
```

Comment:
I manually verified write permissions are the one needed for files and directories for adding or removing exclusions, they are not needed for checking